cpp/src/parquet/encoder.cc

```diff
@@ -1604,6 +1654,7 @@ class RleBooleanEncoder final : public EncoderImpl, virtual public BooleanEncode
           },
           []() { return Status::OK(); }));
     }
+    unencoded_data_bytes_ += sizeof(bool) * (values.length() - values.null_count());
```

Comment:
I think above we said unencoded_bytes_ wasn't supported for bool?

Reply:
Ah, I forgot to delete this line. (I was supposed to include boolean type but it seems to be non-trivial to deal with input with `bool*`, `uint8_t*` and `arrow::BooleanArray` so I removed changes of boolean types in the end)